app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -0,0 +1,43 @@
+package ar.com.wolox.android.example.ui.login;
+
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+import ar.com.wolox.android.R;
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment;
+
+
+public class LoginFragment extends WolmoFragment<LoginPresenter> implements LoginView {
+
+    @Override
+    public int layout() {
+        return R.layout.fragment_login;
+    }
+
+    @Override
+    public void init() {
+        EditText vEmailAccount = requireActivity().findViewById(R.id.vEmailAccount);
+        EditText vPassword = requireActivity().findViewById(R.id.vPassword);
+        Button vLogin = requireActivity().findViewById(R.id.vLogInButton);
+        Button vSignup = requireActivity().findViewById(R.id.vSignUpButton);
+
+        vLogin.setOnClickListener(view -> {
```

Comment:
You should look in the methods you can override, there should be a setListeners in wolmo. All onClickListener should be inside that method

Reply:
There are a few things to mention about this. I think the best is to talk about it. Please write me after so we can have a call